Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -11113,7 +11105,7 @@ def show_jobs(
         status=None,
         id: Optional[int] = None,
         target: Optional[str] = None
-    ) -> List[ShowJobsAllResultData]:
+    ) -> Union[list[ShowJobsAllResultData], ShowJobsAllResultData]:
```

Comment:
when it isnt a list lets put into a list of one so the return type can always be a list

Reply:
<img width="1023" alt="Screenshot 2024-09-02 at 14 25 37" src="https://github.com/user-attachments/assets/03957441-5da8-4d34-a27f-13b1df417f1b">

Reply:
there seems to be a reason it works this way.